hdfs/client.py

```diff
@@ -602,6 +602,17 @@ def rename(self, hdfs_src_path, hdfs_dst_path):
         self.resolve(hdfs_src_path), hdfs_dst_path
       )
 
+  def make_directory(self, hdfs_path, permission=None):
```

Comment:
Can we rename the argument to `permissions` (for consistency with `set_permissions`)?